specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json

```diff
@@ -1955,6 +2086,101 @@
       },
       "description": "The backup key result, containing the backup blob."
     },
+    "KeyRotationPolicy": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The key policy id.",
+          "readOnly": true
+        },
+        "lifetimeActions": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LifetimeActions"
+          },
+          "description": "Actions that will be performed by Key Vault over the lifetime of a key."
+        },
+        "attributes": {
+          "$ref": "#/definitions/KeyRotationPolicyAttributes",
+          "description": "The key rotation policy attributes."
+        }
+      },
+      "description": "Management policy for a key."
+    },
+    "LifetimeActions": {
+      "properties": {
+        "trigger": {
+          "$ref": "#/definitions/LifetimeActionsTrigger",
+          "description": "The condition that will execute the action."
+        },
+        "action": {
+          "$ref": "#/definitions/LifetimeActionsType",
+          "description": "The action that will be executed."
+        }
+      },
+      "description": "Action and its trigger that will be performed by Key Vault over the lifetime of a key."
+    },
+    "LifetimeActionsTrigger": {
+      "properties": {
+        "timeAfterCreation": {
+          "type": "string",
+          "description": "Time after creation to attempt rotate. It will be in ISO 8601 format."
+        },
+        "timeBeforeExpiry": {
```

Comment:
Can we rename this to `timeBeforeExpiration`, to be consistent with `timeAfterCreation`?

Reply:
As discussed offline, make sure that we use "create" and "expiry" everywhere instead of "creation" and "expiration".

Reply:
Updated